forest/config.py

```diff
@@ -21,13 +21,30 @@
 
 """
 import os
+import string
 import yaml
 from forest.export import export
 
 
 __all__ = []
 
 
+def combine_variables(os_environ, args_variables):
+    """Utility function to update environment with user-specified variables
+
+    .. note: When there is a key clash the user-specified args take precedence
+
+    :param os_environ: os.environ dict
+    :param args_variables: variables parsed from command line
```

Comment:
It'd be nice to be more explicit about the structure - i.e. we're expecting a list of 2-tuples.

Reply:
We are actually expecting anything that goes through the `dict` constructor, right now it's actually a list of 2-list after the `--var` redesign